core/src/main/java/nu/yona/server/subscriptions/service/UserPrivateDto.java

```diff
@@ -34,30 +34,28 @@
 	private final UUID userAnonymizedId;
 	private final UUID namedMessageSourceId;
 	private final UUID anonymousMessageSourceId;
-	private final Set<UUID> buddyIds;
-	private final Function<Set<UUID>, Set<BuddyDto>> buddyIdToDtoMapper;
+	private final Set<BuddyDto> buddies;
 
 	@JsonCreator
 	public UserPrivateDto(@JsonProperty("nickname") String nickname)
 	{
-		this(Optional.empty(), null, nickname, null, null, Collections.emptySet(), Collections.emptySet(), null, null,
+		this(Optional.empty(), null, nickname, null, null, Collections.emptySet(), Collections.emptySet(), null,
 				new VPNProfileDto(null));
 	}
 
 	UserPrivateDto(Optional<LocalDate> lastMonitoredActivityDate, String yonaPassword, String nickname, UUID namedMessageSourceId,
-			UUID anonymousMessageSourceId, Set<GoalDto> goals, Set<UUID> buddyIds,
-			Function<Set<UUID>, Set<BuddyDto>> buddyIdToDtoMapper, UUID userAnonymizedId, VPNProfileDto vpnProfile)

```

Comment:
I'm trying to remember where the ``buddyIdToDtoMapper`` was for. I think it was because the mapping couldn't be made upfront, at that time. Do you have any idea?